src/Adapter/Language/LanguageActivator.php

```diff
@@ -60,8 +60,8 @@ private function setActive($langId, $status)
     {
         $lang = new Language((int) $langId);
 
-        if ((int) $lang->active !== (int) $status) {
-            $lang->active = (int) $status;
+        if ($lang->active !== $status) {
```

Comment:
So you assume that everything is correctly casted to `int` 😅
That's optimistic!

Reply:
No, it's now correctly cast in bool :)

Reply:
`active` is a boolean.

For `status`, it's a boolean. Two calll from this class and it's `true` and `false`.